python/ray/tune/commands.py

```diff
@@ -21,6 +23,8 @@
 
 logger = logging.getLogger(__name__)
 
+EDITOR = os.getenv('EDITOR', 'vim')
```

Comment:
```suggestion
EDITOR = os.getenv("EDITOR", "vim")
```
I think we generally use double-quotes. Does the linter not error on this? @richardliaw